iothub/device/devdoc/Convention-based operations.md

```diff
@@ -2,6 +2,12 @@
 
 #### Common
 
+```diff
+public class ClientOptions {
++    public PayloadConvention PayloadConvention { get; set; }
```

Comment:
Shouldn't this be readonly?

Reply:
Good point, yeah, should be. I'll add this to a constructor in a follow-up PR.

Reply:
Readonly doesn't mean it has to be a a ctor. It simply must be set on initialization, and can't be changed after.

`public PayloadConvention PayloadConvention { get; }`

Reply:
But the user needs to be able to set it on initialization =>so it needs to be initialized via a constructor. Is there a different way to do this?